### PR TITLE
update(readme): add new paper and talk references

### PR DESCRIPTION
**Pull request by josecm**

### CODE REVIEW

**Review by danielRep — APPROVED** [review]

File: README.md
Comment (sandro2pinto) [inline README.md:1]: I would suggest replacing the link for the "Shedding Light" paper with the one from arxiv:
https://arxiv.org/abs/2303.11186

**Review by sandro2pinto — CHANGES_REQUESTED** [review]
I would suggest replacing the link for the "Shedding Light" paper with the one from arxiv:
https://arxiv.org/abs/2303.11186

**Comment by josecm** [comment]
> https://arxiv.org/abs/2303.11186

Fixed!

**Review by danielRep — APPROVED** [review]

**Review by sandro2pinto — APPROVED** [review]
All good to me.

**Pull request merged**